### PR TITLE
Few small improvements and doc fixes

### PR DESCRIPTION
**Pull request by jhrcek**
Nice package.
I tried it to rewrite few bash scripts and it feels good :)
Here's bunch of small fixes in docs and few random hlint fixes + exposing one function from Shh module

### CODE REVIEW

File: shh/src/Shh.hs [hunk 1]
```diff
@@ -53,6 +53,7 @@ module Shh
     , captureEndBy
     , captureEndBy0
     , captureLines
+    , captureWords
```
Comment (jhrcek) [inline shh/src/Shh.hs:56]: It feels weird having to import Shh.Internal just to get this function, so exposing it from the "main" module makes sense to me.

File: shh-extras/src/Shh/Prompt.hs [hunk 2]
```diff
@@ -9,7 +9,9 @@ import Data.Time
 import Data.List
 
 -- | The type of GHCi prompt functions
-type PromptFn = [String] -> Int -> IO String
+type PromptFn = [String] -- ^ names of the modules currently in scope
```
Comment (jhrcek) [inline shh-extras/src/Shh/Prompt.hs:12]: I was just confused about what these arguments mean, so I looked it up in ghc user guide and copied the explanation here.

**Review by luke-clifton — APPROVED** [review]

**Comment by luke-clifton** [comment]
Thanks for that, and I'm glad you like it! I'll make a point release with these changes soon.

**Comment by jhrcek** [comment]
Thank you, I appreciate you released to hackage :heart:

**Pull request merged**